charts/openmetadata/templates/check-db-migrations-job-hook.yaml

```
@@ -7,6 +7,13 @@ metadata:
     "helm.sh/hook": pre-upgrade 
     "helm.sh/hook-weight": "-1" # we use a smaller weight so it's created before the job  
     "helm.sh/hook-delete-policy": hook-succeeded, hook-failed
+    {{- if .Values.annotations }}
```

Let's move this piece of code in `_helpers.tpl` file. and refactor all the files.

```suggestion
    {{- define "OpenMetadata.annotations" -}}
    {{- if .Values.annotations }}
    {{- range $key, $val := .Values.annotations }}
    {{ $key }}: {{ $val | quote }}
    {{- end}}
    {{- end}}
    {{- end }}
```
And then, all the Kubernetes Objects should refer to this by calling
`{{- include "OpenMetadata.annotations" . | nindent 4 }}`

---

added {{- include "OpenMetadata.annotations" . | nindent 4 }}